src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,598 @@
+
+`shadowMode: async` writes to both databases.
+
+- The LMS sends application requests to the source of truth and target database in asynchronous threads, and waits only for the source of truth to respond.
+- Results from the source of truth are returned to the application.
+
+You can use this mode to confirm that your queries succeed on CockroachDB without verifying performance or correctness.
+
+### `sync`
+
+<img src="{{ 'images/v23.2/migrations/lms_sync.svg' | relative_url }}" alt="MOLT LMS shadowing mode - sync" />
+
+`shadowMode: sync` writes to both databases.
+
+- The LMS sends application requests to the source of truth and the target database, and waits for each to respond.
+- Results from the source of truth are returned to the application.
+- Results from the non-source of truth are discarded.
+
+You can use this mode to perform an [immediate cutover](#immediate-cutover).
+
+### `strict-sync`
+
+<img src="{{ 'images/v23.2/migrations/lms_strict-sync.svg' | relative_url }}" alt="MOLT LMS shadowing mode - strict-sync" />
+
+`shadowMode: strict-sync` writes to both databases and enforces correctness on both databases.
+
+- The LMS sends application requests to the source of truth and the target database, and waits for each to respond.
+- Results from the source of truth are returned to the application.
+- If the non-source of truth returns an error, that error is returned instead of the result from the source of truth.
+
+You can use this mode to perform an [immediate cutover](#immediate-cutover).
+
+## Perform a cutover
+
+### Consistent cutover
+
+A consistent cutover maintains data consistency with [minimal downtime]({% link {{ page.version.version }}/migration-overview.md %}#minimal-downtime). When using the LMS, consistent cutover is handled using the [`molt-lms-cli`](#molt-lms-cli) commands `cutover consistent begin` and `cutover consistent commit`, between which downtime occurs. The amount of downtime depends on the maximum duration of any transactions and queries that need to complete, and the time it takes for replication to catch up from the source to the target database.
+
+{% comment %}
+For more information about the consistent cutover approach, see [Migration Strategy: Live Migration]({% link {{ page.version.version }}/migration-strategy-live-migration.md %}).
+{% endcomment %}
+
+To perform a consistent cutover with the LMS:
+
+{{site.data.alerts.callout_info}}
+These steps assume you have already followed the overall steps to [prepare for migration]({% link {{ page.version.version }}/migration-overview.md %}#prepare-for-migration). In particular, [update your schema and application queries]({% link {{ page.version.version }}/migration-overview.md %}#update-the-schema-and-queries) to work with CockroachDB.
+{{site.data.alerts.end}}
+
+1. [Configure the LMS](#configuration) with your deployment details, and follow our [security recommendations](#security).
```

Comment:
cc @otan @cotedm do we need to include technical details, especially the `_crdb_proxy_settings` table?

Reply:
I'd prefer not to document this in the docs website. However, having it in the knowledge base for TSE or in a runbook internally I think is valuable. In general, customers shouldn't be messing with these internal details and if it's needed for troubleshooting, we should be holding their hand IMO

Reply:
Ack, will document it in the internal runbook.